Registry/Registry.NMM

```diff
@@ -1932,6 +1932,7 @@ package ltng_none           lightning_option==0            -       -
 package ltng_crm_PR92w      lightning_option==1            -       -
 package ltng_crm_PR92z      lightning_option==2            -       -
 package ltng_cpm_PR92z      lightning_option==11           -       -
+#
```

Comment:
@BarryHLynn 
Barry,
The only change here is a new line with a comment character. Remove it.